terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -149,26 +149,22 @@ resource "kubernetes_cron_job" "workflow_manager" {
     }
   }
   spec {
-    schedule                  = "*/10 * * * *"
-    failed_jobs_history_limit = 3
+    schedule                      = "*/10 * * * *"
+    concurrency_policy            = "Forbid"
+    successful_jobs_history_limit = 5
+    failed_jobs_history_limit     = 3
     job_template {
       metadata {}
       spec {
         template {
           metadata {}
           spec {
             container {
-              name  = "facilitator"
+              name  = "workflow-manager"
               image = "${var.container_registry}/${var.workflow_manager_image}:${var.workflow_manager_version}"
-              # Write sample data to exercise writing into S3.
               args = [
-                "generate-ingestion-sample",
-                "--s3-use-credentials-from-gke-metadata",
-                "--aggregation-id", "fake-1",
-                "--batch-id", "eb03ef04-5f05-4a64-95b2-ca1b841b6885",
-                "--date", "2020/09/11/21/11",
-                "--facilitator-output", "s3://${var.ingestion_bucket}",
-                "--pha-output", "/tmp/sample-pha"
+                "--k8s-namespace", var.kubernetes_namespace,
+                "--input-bucket", "lotophagi-ingestion-bucket",
```

Comment:
I'm guessing `lotophagi-ingestion-bucket` was created manually? Between this and the `prio-workflow-manager` name above (which doesn't exist in [DockerHub](https://hub.docker.com/orgs/letsencrypt/repositories), this module won't work in any env but yours. If it helps keep you moving we should merge this, though.

Reply:
I've published my Docker image to `prio-workflow-manager`, and changed this to refer to a variable that exists in the .tf (though I'm not sure it's the *right* variable).

Reply:
It won't be, because it'll be an S3 bucket and your code only supports GCS buckets for now. But that's OK: it'll deploy just fine, and then the jobs will fail, which just generates noise in the GCP console.